chart/kubeapps/values.yaml

```diff
@@ -2059,14 +2065,6 @@ kubeappsapis:
           ## @param kubeappsapis.pluginConfig.kappController.packages.v1alpha1.defaultAllowDowngrades Default policy for allowing applications to be downgraded to previous versions
           ## ref: https://carvel.dev/kapp-controller/docs/latest/package-consumer-concepts/#downgrading
           defaultAllowDowngrades: false
-    flux:
-      packages:
-        v1alpha1:
-          ## @param kubeappsapis.pluginConfig.flux.packages.v1alpha1.defaultUpgradePolicy Default upgrade policy generating version constraints
-          ## enum: [ "major", "minor", "patch", "none" ]
-          defaultUpgradePolicy: none
-            ## @param kubeappsapis.pluginConfig.flux.packages.v1alpha1.userManagedSecrets Default policy for handling repository secrets, either managed by the user or by kubeapps-apis
-          userManagedSecrets: true
```

Comment:
Is this removal correct @antgamdia ?
Didn't you add `userManagedSecrets` recently to the chart?

Reply:
No, no, it is not correct, but it is normal as we didn't synced this change upstream. We have just to revert those dev changes back, but couldn't get to it yesterday :S